bionty/__init__.py

```diff
@@ -68,10 +68,18 @@
 
    Read the guides:
 
-   - `Access public ontologies <https://lamin.ai/docs/public-ontologies>`__
+   - :doc:`docs:public-ontologies`
    - :doc:`docs:bio-registries`
 
-   For more background on how public ontologies are accessed, see `bionty.base`.
+Bionty base
+===========
+
+Bionty also offers a standalone version `bionty-base` that does not require an active lamindb instance.
+Import it by running:
+
+>>> import bionty.base as bt_base
```

Comment:
If you wanna add this, I'd suggest `btb`. But given this is guerllia use, not suggesting a pattern at all is maybe better.